Buildings/ThermalZones/Detailed/Validation/BESTEST/UsersGuide.mo

```diff
@@ -81,9 +87,11 @@ in the table after the plots below.
 </p>
 <!-- table start: load data -->
 <table border = \"1\" summary=\"Annual load\">
-<tr><td colspan=\"8\"><b>Annual heating load (MWh)</b></td></tr>
+<tr><td colspan=\"10\"><b>Annual heating load (MWh)</b></td></tr>
```

Comment:
Could these limits also be included in the plots of the UserGuide to show whether we satisfy them, or when creating this table, color code the cells green or red whether we satisfy or violate them?